### ISSUE TITLE
Conflict between nginx and mailinabox?

### ISSUE THREAD

**Issue by binarykitchen** [RomainFallet/server-setup#1]
Hi there,

Nice configs and scripts here. Maybe I will give them a try. 

Only one question: mailinabox already comes with predefined nginx configs. Wouldn't your customized ones somehow overlap theirs and break it?

**Comment by RomainFallet** [comment]
Yes, totally, Nginx scripts are not meant to be run on the same machine you installed Mailinabox but can be useful if you want to set up another machine with web hosting capabilities.

**Comment by binarykitchen** [comment]
How did you do this?

**Comment by RomainFallet** [comment]
What do you mean?

**Comment by binarykitchen** [comment]
@RomainFallet your server-setup installs mail-in-a-box, and also other components using nginx, like certbot. How did you make sure both won't conflict with each other?
